### PR TITLE
Update features html

### PR DESCRIPTION
**Pull request by sripor62**
## 👨‍💻 Changes proposed
I have updated the features.html and added a google fonts link to the base.html.
So now features will look as in the screenshot.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Screenshot (215)

### CODE REVIEW

**Comment by KKhushhalR2405** [comment]
@sripor62 Your PR have some conflicts. Can you please resolve them. You can refer [here](https://youtu.be/JtIX3HJKwfo) to learn about resolving merge conflicts.

**Comment by KKhushhalR2405** [comment]
@sripor62 The file is giving some errors.
image

**Comment by KKhushhalR2405** [comment]
@sripor62 The file is giving some errors.
image

File: templates/home/Features/features.html [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+<h4 class="fea" style="margin-left:10px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   In-depth concept coverage</h4>
+<h4 class="fea" style="margin-right:86px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   One step solution</h4>
+<h4 class="fea" style="margin-right:68px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   24*7 Doubt solving</h4>
+<h4 class="fea" style="margin-right:90px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   1 on 1 Mentoring</h4>
+<h4 class="fea" style="margin-right:88px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   Project Guidance</h4>
+<h4 class="fea" style="margin-right:45px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   Certification Included</h4>
+<br><br>
+
+{% endblock content %}
 <div class="heading text-center display-3">CodingEasy</div><br>
```
Comment (KKhushhalR2405) [inline templates/home/Features/features.html:20]: You should delete lines 19 to 31. It should work fine then.

**Review by KKhushhalR2405 — CHANGES_REQUESTED** [review]

File: templates/home/Features/features.html (hunk 1 above)
Comment (sripor62) [reply templates/home/Features/features.html]: Did it

**Review by KKhushhalR2405 — APPROVED** [review]

**Comment by KKhushhalR2405** [comment]
Great Work @sripor62

**Comment by KKhushhalR2405** [comment]
LGTM @arpit456jain.

**Pull request merged**